pkg/ctl/create/cluster.go

```diff
@@ -350,18 +348,6 @@ func doCreateCluster(rc *cmdutils.ResourceCmd, params *createClusterCmdParams) e
 			return err
 		}
 
-		// add default storage class only for version 1.10 clusters
-		if meta.Version == "1.10" {
-			// --storage-class flag is only for backwards compatibility,
-			// we always create the storage class when --config-file is
-			// used, as this is 1.10-only
-			if params.addonsStorageClass || rc.ClusterConfigFile != "" {
-				if err = ctl.AddDefaultStorageClass(clientSet); err != nil {
```

Comment:
Actually we should also delete the function itself :)

Reply:
Thanks, had missed this. Now removed.